plugins/woocommerce/includes/wc-formatting-functions.php

```diff
@@ -13,6 +13,8 @@
 
 defined( 'ABSPATH' ) || exit;
 
+//TODO: Once WooCommerce requires PHP 7.4, the "$x = $x ?? ''" constructs can be replaced with "$x ??= ''".
```

Comment:
Couple of PHPCS violations here:

```
WARNING | Comment refers to a TODO task (Generic.Commenting.Todo.TaskFound)
ERROR | No space found before comment text (Squiz.Commenting.InlineComment.NoSpaceBefore)
```

I'm assuming we only need to clear the error and not the warning:

```suggestion
// TODO: Once WooCommerce requires PHP 7.4, the "$x = $x ?? ''" constructs can be replaced with "$x ??= ''".
```

Reply:
Committed the suggestion, will see if that unblocks merge.